common/djangoapps/course_modes/views.py

```diff
@@ -39,7 +38,7 @@ class ChooseModeView(View):
     """
 
     @method_decorator(transaction.non_atomic_requests)
-    def dispatch(self, *args, **kwargs):        # pylint: disable=missing-docstring
+    def dispatch(self, *args, **kwargs):  # pylint: disable=missing-docstring
```

Comment:
Would it be possible to add a docstring for this method and remove the pylint override?